new_dawn_server/users/api/resources.py

```diff
@@ -276,7 +277,8 @@ class Meta:
             'age': ALL_WITH_RELATIONS,
             'review_status': ALL_WITH_RELATIONS,
         }
-        queryset = Profile.objects.all()
+        profiles_count = Profile.objects.all().count()
```

Comment:
Let's get the Profile.objects.all() first `profiles = Profile.objects.all()`, and then get its count and queryset from `profiles` to avoid fetching db twice.